### PR TITLE
Makefile adjustments

### PR DESCRIPTION
**Pull request by fedeztk**
Small improvements in the Makefile:
- added clean and uninstall options
- build without the need of `mv`, using go builtin -o flag

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ git clone git@github.com:BetaPictoris/dict.git      # Clone the repository
 cd dict                                             # Change into the repository's directory
 make                                                # Build Wiki CLI
 sudo make install                                   # Install Wiki CLI to "/usr/local/bin" with the mode "755"
+# uninstall:
+sudo make uninstall
```
Comment (BetaPictoris) [inline docs/readme.md:36]: I'd change this to fit in with the rest of the comments, or put it in another readme section.

**Review by BetaPictoris — CHANGES_REQUESTED** [review]

File: docs/readme.md (hunk 1 above)
Comment (fedeztk) [reply docs/readme.md:36]: Something like:
`sudo make uninstall #uninstall`
would be fine? Or is it better to just move it in its own section like:
## Uninstall
`sudo make uninstall`

Comment (BetaPictoris) [reply docs/readme.md:36]: I think the first option, with the comment on the left looks good.

**Review by BetaPictoris — APPROVED** [review]
Thank you!

**Pull request merged**